reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md

```diff
@@ -0,0 +1,345 @@
+- Fix `pwsh -version` and built-in help for `pwsh.exe` to align with other native tools. (#4958 & #4931) (Thanks @iSazonov)
+- Make invalid argument error messages for `-File` and `-Command` consistent and make exit codes consistent with Unix standards (#4573)
+- Add `-WindowStyle` parameter on Windows. (#4573)
+
+## Moved from .NET Framework to .NET Core
+
+TODO
+
+## Support for macOS and Linux
+
+PowerShell now supports macOS and Linux, including:
+
+TODO
+
+We also made a number of changes to PowerShell in order to make it work better with non-Windows ecosystems.
+Some of these are breaking changes which also affect Windows.
+Others are only present or applicable in non-Windows installations of PowerShell Core.
+
+TODO: organize
+- Add support for native command globbing on Unix platforms.
+- `more` function respects the Linux `$PAGER`; defaults to `less`
+ This means you can now use wildcards with native binaries/commands (e.g. `ls *.txt`). (#3463)
+- Escape trailing backslash when dealing with native command arguments. (#4965)
+- Ignore the `-ExecutionPolicy` switch when running PowerShell on non-Windows platforms because script signing is not currently supported. (#3481)
+- Fix ConsoleHost to honour `NoEcho` on Unix platforms. (#3801)
+- Fix `Get-Help` to support case insensitive pattern matching on Unix platforms. (#3852)
+- `powershell` man-page added to package
+
+### Logging
+
+On macOS, PowerShell uses the native `os_log` APIs to log to Apple's [unified logging system][os_log].
+On Linux, PowerShell uses [Syslog][], a ubiquitous logging solution.
+
+### Filesystem
+
+A number of changes have been made on macOS and Linux to support filename characters not traditionally supported on Windows:
+
+- Paths given to cmdlets are now slash-agnostic (both / and \ work as directory separator)
+- XDG Base Directory Specification is now respected and used by default.
+ This means that the Linux/macOS profile path is located at `~/.config/powershell/profile.ps1`,
+ the history save path is located at `~/.local/share/powershell/PSReadline/ConsoleHost_history.txt`,
+ and the user module path is located at `~/.local/share/powershell/Modules`.
+- Enable support of folders and files with colon in name on Unix. (#4959)
+- Script names or full paths can have commas. (#4136) (Thanks to @TimCurwick!)
+- Fix detection of whether `-LiteralPath` was used to suppress wildcard expansion for navigation cmdlets. (#5038)
+- Update `Get-ChildItem` to be more in line with the way that the *nix `ls -R` and the Windows `DIR /S` native commands handle symbolic links to directories during a recursive search.
+ Now, `Get-ChildItem` returns the symbolic links it encountered during the search, but it won't search the directories those links target. (#3780)
+
+### Case sensitivity
+
+Generally speaking, PowerShell tries to be case insensitive, while macOS and Linux tend to be case sensitive
```

Comment:
This isn't correct.  macOS filesystem seems to be case insensitive by default but can be case sensitive.